sklearn/svm/tests/test_svm.py

```diff
@@ -1059,16 +1105,17 @@ def test_svc_bad_kernel():
         svc.fit(X, Y)
 
 
-def test_timeout():
+def test_libsvm_convergence_warnings():
```

Comment:
I changed the name of the test so that it is consistent with the test for liblinear (`test_linear_svm_convergence_warnings`)